test/e2e/tests/02-fields-initialized.js

```diff
@@ -242,9 +242,14 @@ module.exports = {
     expectInputToNotBeSelected(client, 'input[name="isMedicaidEligible-1"]');
     expectInputToNotBeSelected(client, 'input[name="isEnrolledMedicarePartA-0"]');
     expectInputToNotBeSelected(client, 'input[name="isEnrolledMedicarePartA-1"]');
+
+    client
```

Comment:
I was able to get these tests to pass with the following code (replacing what you have starting at line 246):
```javascript
client.click('input[name="isEnrolledMedicarePartA-0"]');

expectValueToBeBlank(client, 'select[name="medicarePartAEffectiveMonth"]');
expectValueToBeBlank(client, 'select[name="medicarePartAEffectiveDay"]');
expectValueToBeBlank(client, 'input[name="medicarePartAEffectiveYear"]');
```

Try that and see if the tests pass for you. You can run an individual test by doing the following things:
1. Make sure your `npm run watch` command is running in one terminal window
2. Run `selenium-server -p 4444` in another terminal window
3. Run `./node_modules/nightwatch/bin/nightwatch -c test/e2e/nightwatch.js --test test/e2e/tests/02-fields-initialized.js` to run an individual file or `npm run tests:e2e` to run them all but that takes awhile.

If setting up the tests to run locally is causing you problems don't worry for now, just push this change and Travis should run it.